.babelrc

```diff
@@ -1,4 +1,5 @@
 {
+  "sourceType": "unambiguous",
```

Comment:
Similarly to the legacy babel config, polyfills will be added here too, if needed (in case a certain feature is not supported in "modern" browsers)

Reply:
(just linking myself to https://babeljs.io/docs/en/options#sourcetype `unambiguous` here)